src/Security/Authorization/Core/src/LoggingExtensions.cs

```diff
@@ -15,11 +15,11 @@ static LoggingExtensions()
         {
             _userAuthorizationSucceeded = LoggerMessage.Define(
                 eventId: new EventId(1, "UserAuthorizationSucceeded"),
-                logLevel: LogLevel.Information,
+                logLevel: LogLevel.Debug,
                 formatString: "Authorization was successful.");
             _userAuthorizationFailed = LoggerMessage.Define<string>(
                 eventId: new EventId(2, "UserAuthorizationFailed"),
-                logLevel: LogLevel.Information,
+                logLevel: LogLevel.Debug,
                 formatString: "Authorization failed. {0}");
```

Comment:
The failure scenario seems to be providing more data than they reported in the issue. Looks like that was a 5.0 change?

If the failure case is providing useful details now then we should keep that at the existing log level.

Reply:
It depends on the code path really, but sure we can keep the failure as Information and the success as Debug, thoughts @blowdart ?  There's potentially more information if requirements failed as opposed to explicit Fail being called

Reply:
Darn it. Ok yes, keep fail as informational then.